### PR TITLE
Update the CMake code

### PR DESCRIPTION
**Pull request by juan-lunarg**
Required tweaking known_good.json and a couple C++ changes

This PR requires this to be merged in first:
https://github.com/LunarG/VulkanTests/pull/398

### CODE REVIEW

**Comment by juan-lunarg** [comment]
> CI Vulkan-ExtensionLayer build # 538 passed.

CI passes now that my change to VulkanTests is in.

**Comment by juan-lunarg** [comment]
@jeremyg-lunarg this is ready for review now.

File: scripts/known_good.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
             "build_dir": "glslang/build",
             "install_dir": "glslang/build/install",
             "commit": "d1517d64cfca91f573af1bf7341dc3a5113349c0",
-            "prebuild": [
-                "{python} update_glslang_sources.py"
```
Comment (jeremyg-lunarg) [inline scripts/known_good.json:19]: yay, this part has been problematic for a while

**Review by jeremyg-lunarg — APPROVED** [review]
LGTM, thanks for doing this!

**Comment by christophe-lunarg** [comment]
Any plan to use CMake for the Android build on this repository?

**Comment by juan-lunarg** [comment]
> Any plan to use CMake for the Android build on this repository?

Yes. I my plan is to get CMake working for all the Android builds. Not just VVL.

**Pull request merged**